src/SFA.DAS.EmployerAccounts.Jobs/StartupJobs/CreateReadStoreDatabaseJob.cs

```diff
@@ -46,6 +49,8 @@ public async Task Run()
                 }
             };
 
+            _logger.LogInformation("Creating ReadStore database and collection if they don't exist");
+
             await _documentClient.CreateDatabaseIfNotExistsAsync(database);
             await _documentClient.CreateDocumentCollectionIfNotExistsAsync(UriFactory.CreateDatabaseUri(database.Id), documentCollection);
```

Comment:
Similarly, it would be nice to log whether the collection was created or not.

Reply:
As above, we can't see if this created the collection or just found it